samples/csharp_dotnetcore/language-generation/05.a.multi-turn-prompt-with-language-fallback/MultiTurnMultiLingualPromptBot.csproj

```diff
@@ -5,11 +5,26 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Bot.Builder" Version="4.14.1" />
-    <PackageReference Include="Microsoft.Bot.Builder.Dialogs" Version="4.14.1" />
-    <PackageReference Include="Microsoft.Bot.Builder.Integration.AspNet.Core" Version="4.14.1" />
-    <PackageReference Include="Microsoft.Bot.Builder.LanguageGeneration" Version="4.14.1" />
-    <PackageReference Include="Microsoft.Bot.Builder.Dialogs.Adaptive" Version="4.14.1" />
+    <PackageReference Include="Microsoft.Bot.Builder" Version="4.15.0-daily.20211006.271232.c20cc39" />
+    <PackageReference Include="Microsoft.Bot.Builder.Dialogs" Version="4.15.0-daily.20211006.271232.c20cc39" />
+    <PackageReference Include="Microsoft.Bot.Builder.Integration.AspNet.Core" Version="4.15.0-daily.20211006.271232.c20cc39" />
+    <PackageReference Include="Microsoft.Bot.Builder.LanguageGeneration" Version="4.15.0-daily.20211006.271232.c20cc39" />
+    <PackageReference Include="Microsoft.Bot.Builder.Dialogs.Adaptive" Version="4.15.0-daily.20211006.271232.c20cc39" />
+  </ItemGroup>
+
+  <ItemGroup>
```

Comment:
Why is this `ItemGroup` added to all csproj files?

Reply:
These `ItemGroup` for the `.csproj` files were added due to an issue (#3438) found, that when the bots were deployed to Azure, they fail to find the `.lg` files (`e.g.: AdapterWithErrorHandler.lg`).
By adding the ItemGroup to copy the file to the built folder, we ensure they are uploaded to Azure when the bot get's deployed.